packages/utils/telemetry-utils/src/sampledTelemetryHelper.ts

```diff
@@ -58,7 +58,14 @@ interface Measurements {
  * @internal
  */
 export class SampledTelemetryHelper implements IDisposable {
-	disposed: boolean = false;
+	private _disposed: boolean = false;
+
+	/**
+	 * {@inheritDoc @fluidframework/core-interfaces#IDisposable.disposed}
+	 */
+	public get disposed(): boolean {
```

Comment:
This was previously externally mutable. Updated to use a private field and public getter to ensure external users can't mess with its `disposed` state.